src/garage/sampler/utils.py

```diff
@@ -13,7 +13,8 @@ def rollout(env,
             max_path_length=np.inf,
             animated=False,
             speedup=1,
-            deterministic=False):
+            deterministic=False,
+            accum_context=False):
```

Comment:
I think we should create a separate rollout function that has accum_context, and keep it inside the file `pearl_sampler.py` if accumulation context is only used in the context of PEARL

Reply:
i agree. @krzentner would have a better idea of what to do.